atum/src/test/scala/za/co/absa/atum/ControlInfoToJsonSerializationSpec.scala

```diff
@@ -80,39 +80,116 @@ class ControlInfoToJsonSerializationSpec extends AnyFlatSpec with Matchers {
     )
   )
   val version = BuildProperties.buildVersion
-  val exampleInputJson: String = "{\"metadata\":{\"sourceApplication\":\"FrontArena\",\"country\":\"ZA\"," +
-    "\"historyType\":\"Snapshot\",\"dataFilename\":\"example.dat\",\"sourceType\":\"\"," +
-    "\"version\":1,\"informationDate\":\"01-01-2017\",\"additionalInfo\":{\"key1\":\"value1\",\"key2\":\"value2\"}}," +
-    "\"checkpoints\":[{\"name\":\"Source\"," +
-    "\"software\":\"Atum\",\"version\":\""+ version + "\"," +
-    "\"processStartTime\":\"01-01-2017 08:00:00\"," +
-    "\"processEndTime\":\"01-01-2017 08:00:00\",\"workflowName\":\"Source\",\"order\":1," +
-    "\"controls\":[{\"controlName\":\"pvControlTotal\",\"controlType\":\"type.aggregatedTotal\"," +
-    "\"controlCol\":\"pv\",\"controlValue\":\"32847283324.324324\"},{\"controlName\":\"recordCount\"," +
-    "\"controlType\":\"type.Count\",\"controlCol\":\"id\",\"controlValue\":243}]},{\"name\":\"Raw\"," +
-    "\"software\":\"Atum\",\"version\":\""+ version + "\"," +
-    "\"processStartTime\":\"01-01-2017 08:00:00\",\"processEndTime\":\"01-01-2017 08:00:00\"," +
-    "\"workflowName\":\"Raw\",\"order\":2,\"controls\":[{\"controlName\":\"pvControlTotal\"," +
-    "\"controlType\":\"type.aggregatedTotal\",\"controlCol\":\"pv\",\"controlValue\":\"32847283324.324324\"}," +
-    "{\"controlName\":\"recordCount\",\"controlType\":\"type.Count\",\"controlCol\":\"id\"," +
-    "\"controlValue\":243}]}]}"
+  val software = BuildProperties.projectName
+  val exampleInputJson: String = s"""{
```

Comment:
If you do not like this styling I am happy to revert. It is just easier for me to read.